openshift/README.md

```diff
@@ -0,0 +1,103 @@
+3.2.x or greater and the `restricted` SCC without using a workaround. This is due to a longstanding bug in Helm, which
+you can see [here](https://github.com/helm/helm/issues/9136). You can read more about this in the [Usage](#usage)
+section.
+
+## Compatibility
+
+We test against the following Openshift versions, and guarantee compatibility with:
+
+| Openshift Version |     Supported      |
+|-------------------|--------------------|
+| 4.10.x            | :white_check_mark: |
+
+Any version not explicitly marked in the table above is not tested, and we cannot guarantee compatibility.
+
+## Usage
+
+You will find in this repository a sample `values.yaml` file to get you start on Openshift
+[here](/openshift/values.yaml).
+
+Before proceeding, make sure you've fulfilled all the requirements as described in the [README](/README.md), namely that
+you have Helm installed, and you've added the chart's repository.
+
+Finally, due to a longstanding bug in Helm (see [here](https://github.com/helm/helm/issues/9136)), the installation
+instructions are a bit different depending on your Helm version. To find out, quickly run:
+
+```shell
+helm version
+```
+
+For example, if your version was "3.9.0", you would get something like this:
+
+```shell
+$ helm version
+version.BuildInfo{Version:"v3.9.0", GitCommit:"7ceeda6c585217a19a1131663d8cd1f7d641b2a7", GitTreeState:"clean", GoVersion:"go1.17.5"}
+```
+
+If your version is greater than or equal to 3.2.0, refer to [this section](#helm-32x-and-greater). If it's lower, refer
+to [this section](#helm-313-or-lower).
+
+### Helm 3.1.3 or lower
+
+If you're using Helm 3.1.3 or lower, you can simply install the chart as you normally would. Copy
+the [values.yaml](/openshift/values.yaml) locally, or merge the values with your own values file.
+
+> Make sure you've logged into your Openshift cluster, and have selected a project to deploy the chart into. You can
+> login using `oc login`, and create a new project via `oc new-project myProject`.
+
+```shell
+helm install quick-test camunda/camunda-platform --skip-crds --values values.yaml
+```
+
+You can verify the installation as described in the [README.md](/README.md).
+
+### Helm 3.2.x and greater
+
+Because Helm 3.2.x or greater cannot unset default values in sub-charts, we have two options: allow usage of the
+`anyuid` or `nonroot` SCC, or use `kustomize` as a chart post-renderer.
+
+#### anyuid or nonroot SCC
+
+Under the `restricted` SCC, these charts (`elasticsearch`, `bitnami/keycloak`, and `bitnami/postgresql`) would fail to
+deploy. The simplest method is thus to allow the user/service account which will deploy your chart to use the `anyuid`
+or `nonroot` SCC. This will let sub-charts which define arbitrary UIDs/GIDs use these IDs.
+
+For example, if you will use the user `deployer` to deploy the chart but still want to restrict running to nonroot
+users:
+
+```shell
+oc adm policy add-scc-to-user nonroot deployer
+```
+
+#### kustomize post-renderer
```

Comment:
:wrench: We should add a warning that this is all-or-nothing: If a user sets, for example, fsGroup of the postgres statefulset, using the post-renderer will remove that value. The user would have to adjust the kustomization.yaml.

Reply:
That's a good point. I need to update the docs to highlight a few things:

1. If you use the post-renderer, you need to __always__ use the post-renderer, even on upgrade, otherwise the values will come back.
2. You're correct that there's a one-to-one mapping between the values you want to unset and the patches. I'm not sure how to best express that, but it's important that we highlight it so it's easy to customize. I'll try something.